.env.development

```diff
@@ -46,6 +46,8 @@ FEATURE_ENABLE_AUTO_APPLIED_LICENSES='true'
 LEARNING_TYPE_FACET='true'
 LEARNER_SUPPORT_URL='https://support.edx.org/hc/en-us'
 LEARNER_SUPPORT_SPEND_ENROLLMENT_LIMITS_URL='http://edx.org'
+LEARNER_SUPPORT_ABOUT_LIMITS_URL='https://business-support.edx.org/hc/en-us/articles/14473635245079-Learner-Spending-Limits'
```

Comment:
Is it okay to put the exact help links in, or do we prefer placeholder links we're more sure will never change?

Reply:
Good observation!  I don't think we need a new configuration setting in this case.